### PR TITLE
28 current method for loading the bot api key is archaic

### LINKED ISSUE #28: current method for loading the bot api key is archaic
We currently use a utils.py script for loading the API key, whereas we could point to the env variables from the OS or from a locally created variable. 
Enable support for that and remove the old method we used.

### PR DESCRIPTION
**Pull request by elroykanye**
fixes #28 

Changes coming into this repo from this branch include:
 - added the python-decouple package, using decouple.config to load the env var for the API key
 - deleted the utils.py file

### CODE REVIEW

File: src/app/constants.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
-from utils import get_api_key
+from decouple import config
```
Comment (elhmn) [inline src/app/constants.py:1]: can we create an `.env.example` file with this text ?
```
TELEGRAM_BOT_API_KEY=<telegram_api_key>
```

File: requirements.txt [hunk 2]
```diff
@@ -1,3 +1,4 @@
 python-telegram-bot
```
Comment (elhmn) [inline requirements.txt:1]: Is this file automatically generated by `python -m pip freeze > requirements.txt` ?

File: src/app/constants.py (hunk 1 above)
Comment (elroykanye) [reply src/app/constants.py:1]: Already did that but excluded it in gitignore so it can remain private

File: requirements.txt (hunk 2 above)
Comment (elroykanye) [reply requirements.txt:1]: Oh its not

File: src/app/constants.py (hunk 1 above)
Comment (elhmn) [reply src/app/constants.py:1]: The `.env.example` should not be in the `.gitignore` only the `.env` should. The `.env.example` is used to show maintainers how to create the `.env` file

File: requirements.txt (hunk 2 above)
Comment (elhmn) [reply requirements.txt:1]: Can you generate that file with that commend, and try to use fixed libraries

File: src/app/constants.py (hunk 1 above)
Comment (elroykanye) [reply src/app/constants.py:1]: ah okay. Thanks, I get it now

File: requirements.txt (hunk 2 above)
Comment (elroykanye) [reply requirements.txt:1]: I will :) thanks for this

**Comment by elroykanye** [comment]
@elhmn , I have done the updates 💯

**Review by Mankavelda — APPROVED** [review]

**Review by elhmn — APPROVED** [review]

**Pull request merged**